6.bulk_Morphology_Elastic_Network/1.train_models/notebooks/1.train_regression_multi_output.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from sklearn.model_selection import LeaveOneOut
</pre><p>Wow, I didn't know this existed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Me either! I think this worked well too... Considering the low sample size t he LeaveOneOut approach was probably the best one</p>